doc/data/messages/f/fixme/details.rst

```diff
@@ -1 +0,0 @@
-You can help us make the doc better `by contributing <https://github.com/PyCQA/pylint/issues/5953>`_ !
```

Comment:
This check is very opinionated right now, please help us make it better by contributing to https://github.com/PyCQA/pylint/issues/2874

 (I can't suggest directly on deleted lines)

Reply:
Okay, `details.rst` file has been recreated with your words.